src/microlite_esp/app.py

```diff
@@ -309,7 +289,7 @@ def continue_create_model(req, res):
     yield from res.awrite(html)
 
 async def read_model_byte_data_to_file(req):
-    f = open(TMP_MODEL_PATH, 'wb')
+    app_manager = AppManager()
     final_index = int(req.headers[b"Content-Length"]) - MODEL_REQUEST_END_LEGTH

```

Comment:
`MODEL_REQUEST_END_LEGTH` has a typo I believe

Reply:
Yep, Will fix it when master willl be up to date